src/main/java/de/rabea/game/Game.java

```diff
@@ -46,7 +27,7 @@ public void play(int boardDimension) {
         }
         finishGame(currentPlayer.mark(), board);
         if (userInterface.playAgain()) {
-            setUpNewGame();
+            gameSetup.setUpNewGame();
```

Comment:
I don't think I understand what you mean here... In this case the `Game` is asking the `GameSetup` class to set up the new game. Or do you mean that `Game` shouldn't know if the user wants to play again or not?